### PR TITLE
Commit von Benedek Kámán

### PR DESCRIPTION
**Pull request by bekaman7**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -22,26 +15,12 @@ GEM
...
-      parser (>= 2.7.1.4)
-    ruby-progressbar (1.10.1)
-    unicode-display_width (1.7.0)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
   rspec
-  rubocop
```
Comment (sislr) [inline Gemfile.lock:44]: Why did you remove rubocop? We use Rubocop as a linter and code formatter and this helps us to improve our code quality and readability.

**Review by sislr — APPROVED** [review]
Nicely done, I like it! 🥇

File: Gemfile.lock (hunk 1 above)
Comment (bekaman7) [reply Gemfile.lock:44]: I haven't touched Gemfile.lock. Since rubocop is not defined as a dependency in Gemfile it was automatically removed from the file when I did "bundle install".
